libs/functions/generate_api.py

```diff
@@ -67,52 +69,55 @@ def _generate_api_for_projections(projection_row):
     projections = _Projections(
         totalHospitalBeds=_hospital_beds,
         ICUBeds=None,
-        cumulativeDeaths=projection_row[rc.CURRENT_DEATHS],
-        peakDeaths=None,
-        peakDeathsDate=None,
-        endDate=_format_date(projection_row[rc.LAST_UPDATED])
-        + timedelta(days=FRAMES * DAYS_PER_FRAME),
     )
     return projections
 
+def _generate_actuals(projection_row, intervention, state):
+    intervention_str = {
+        Intervention.NO_INTERVENTION: 'limited_action',
+        Intervention.FLATTEN: 'stay_at_home',
+        Intervention.INFERRED: 'projected',
+        Intervention.SOCIAL_DISTANCING: 'social_distancing',
+    }[get_can_projection.get_intervention(intervention, state)]
+    return _Actuals(
+        population=projection_row[rc.POPULATION],
+        intervention=intervention_str,
+        cumulativeConfirmedCases=projection_row[rc.CURRENT_CONFIRMED],
+        cumulativeDeaths=projection_row[rc.CURRENT_DEATHS],
+        hospitalBeds = {
```

Comment:
dors this work? does it not have to be _ResourceUtilization

Reply:
It does work!